modules/ROOT/pages/authentication-authorization/privileges-reads.adoc

```diff
@@ -17,10 +17,7 @@ There are three separate read privileges:
 * xref:authentication-authorization/privileges-reads.adoc#access-control-privileges-reads-read[`READ`] - enables the specified properties of the found entities to be read.
 * xref:authentication-authorization/privileges-reads.adoc#access-control-privileges-reads-match[`MATCH`] - combines both `TRAVERSE` and `READ`, enabling an entity to be found and its properties read.
 
-[NOTE]
-====
-For more details about the syntax descriptions, see xref:database-administration/syntax.adoc[].
-====
+For more details about the syntax descriptions, see xref:/authentication-authorization/manage-privileges.adoc#components-of-the-graph-privilege-commands[Components of the graph privilege commands].
```

Comment:
I think this previously linked to the general page about the syntax used for these syntax descriptions, for example what the meaning of things like `{}` or `[]` are. Now you switched so it's no longer doing that. If you want to point to the general section about the components maybe have both links, one for syntax descriptions and one for the components

Reply:
I saw that, but I found it a bit confusing because it was about general syntax and also part of the database management section. But I see your point as well.

Reply:
yeah, before the move to the ops manual the syntax page didn't live in the database management section (it was in the general admin section that had both the database management and the role/user/privilege parts), but it was put there in the move. It is a section shared between privileges/users/roles, databases/aliases as well as indexes/constraints/non-admin show commands (that live in cypher manual) so it's not really clear where it should live. 

I think the idea is to introduce a reference grammar that would also be used in the documentation and when we get to that stage these syntax descriptions would be updated to use that format instead, but until then we're just keeping with this format that we have currently